android/rctmgl/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapView.java

```diff
@@ -428,6 +432,16 @@ public void onStyleLoaded(@NonNull Style style) {
                 createSymbolManager(style);
                 setUpImage(style);
                 addQueuedFeatures();
+
+                mLocalizationPlugin = new LocalizationPlugin(RCTMGLMapView.this, mMap, style);
+                if (mLocalizeLabels) {
+                    try {
+                        mLocalizationPlugin.matchMapLanguageWithDeviceDefault();
+                    } catch (Exception e) {
+                        final String localeString = Locale.getDefault().toString();
+                        Log.w(LOG_TAG, String.format("Could not find matching locale for %s", localeString));
```

Comment:
we might want to use: `com.mapbox.mapboxsdk.log.Logger` here. The advantage is that it's transferred to JS side, via our js Logger, so that would show as a yellow box warning.
https://docs.mapbox.com/android/api/map-sdk/8.2.1/com/mapbox/mapboxsdk/log/Logger.html
https://github.com/react-native-mapbox-gl/maps/blob/master/javascript/utils/Logger.js
```suggestion
                        Logger.w(LOG_TAG, String.format("Could not find matching locale for %s", localeString));
```

Reply:
Got it. That does look a lot better and would be good to get the yellow box warning. I'll commit this suggestion.